releasenotes/notes/remove-less-than-4-python-constraint-b840e7405d0a75fd.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
```

Comment:
could you remove this yaml file from your MR? I think removing python <4 constraint should not be visible to any users, or it should not cause any functionality change and I include only those to the changelog.

Reply:
Gotcha. Done: 0ccebbde1d63a4db4502c2bcc5c1867557b0db97